### ISSUE TITLE
All casaos-cli app-management commands result in 401 error after 0.4.4 upgrade

### ISSUE THREAD

**Issue by devcircus** [IceWhaleTech/CasaOS#1247]
**Describe the bug**

After upgrading to the official 0.4.4 from the 0.4.4 alpha, I can't run any `casaos-cli app-management` commands without getting `401 unauthorized`.

If I revert to 0.4.3, the commands run successfully. Upgrading back to 0.4.4, I get `401 unauthorized` again.
For example, running the following command gives me this error:
```sh
casaos-cli app-management list app-stores -u "localhost:86"
```

**To Reproduce**
1. Upgrade to 0.4.4 from 0.4.4 alpha
2. Run any `casaos-cli app-management` commands
3. See error

**Expected behavior**
Expect to see the results from the command that was run

**Desktop (please complete the following information):**

```
 - Host: Raspberry Pi 4 Model B Rev 1.4
 - OS: Ubuntu 22.04.2 LTS aarch64
 - Browser N/A (cli)
 - Version: 0.4.4
```

**System Time**

```
Fri 2023-07-14 21:25:58 UTC
```

**Logs**

Lots of app-management service errors due to custom stores not being recognized correctly, but can't find any errors directly related to my issue
[casaos-cli app-management service log pastebin](https://pastebin.com/b6E9dm3i)

**Comment by devcircus** [comment]
For the time being, I've reverted to the alpha and everything works correctly.

**Comment by WisdomSky** [comment]
was your issue related to installing third-party appstores?

You can now directly install an appstore from within CasaOS' dashboard. Just click the appstore and you'll find an add source button in there. Paste the .zip link of the appstore and submit.

**Comment by devcircus** [comment]
Like I mentioned, immediately following the update, if I run any app-management command, I get the error. In my case, I ran the update, then tried to list my stores, but got an error. I do have several third party stores installed though. I guess I could try removing the third party stores, then perform the update and re-register the stores. 

I'll try that and report back.

**Comment by tigerinus** [comment]
might be related to the recent security vulnerability fix

**Comment by CorrectRoadH** [comment]
👀 I can't reproduce the problem. But my CLI is the latest that compiles from code. the 0.4.4 might didn't update the CLI binary. So I think it is caused by the version didn't match.
You can wait for the next update, I think it will resolve the problem.

**Comment by tigerinus** [comment]
@LinkLeong please fix the setup/update script to include CasaOS-CLI v0.4.4

**Comment by devcircus** [comment]
No problem waiting for the next update. Thanks for looking into it.

**Comment by rodrigofragadf** [comment]
I'm having the same problem. CasaOS 0.4.4, Debian 12, AMD64. Right after installation I try to run the app-management commands and get the 401 error. Is there any way to fix this?

**Comment by WisdomSky** [comment]
I also hope that this will be fixed as soon as possible. I was working on something that utilizes the cli commands but now can't proceed coz of this issue.

**Comment by CorrectRoadH** [comment]
> I also hope that this will be fixed as soon as possible. I was working on something that utilizes the cli commands but now can't proceed coz of this issue.

🥲It may be a strange Bug. We can't reproduce it.

**Comment by tigerinus** [comment]
> I also hope that this will be fixed as soon as possible. I was working on something that utilizes the cli commands but now can't proceed coz of this issue.

Can you download directly from https://github.com/IceWhaleTech/CasaOS-CLI/releases/tag/v0.4.4 and see if it works?

**Comment by tigerinus** [comment]
> I'm having the same problem. CasaOS 0.4.4, Debian 12, AMD64. Right after installation I try to run the app-management commands and get the 401 error. Is there any way to fix this?

Are you running CLI locally or from a remote, like SSH?

**Comment by WisdomSky** [comment]
I'm currently running official `v0.4.4`

![image](https://github.com/IceWhaleTech/CasaOS/assets/6792172/abadd161-cbb1-4ee3-a528-e9cee633d9e5)

and I'm getting Error 401 every time I access any of the commands:
![image](https://github.com/IceWhaleTech/CasaOS/assets/6792172/78eee158-cf2b-4b4a-96b4-c8efbb0b9a9d)

**Comment by rodrigofragadf** [comment]
> > I'm having the same problem. CasaOS 0.4.4, Debian 12, AMD64. Right after installation I try to run the app-management commands and get the 401 error. Is there any way to fix this?
> 
> Are you running CLI locally or from a remote, like SSH?

I tried running over SSH and locally but both didn't work.  I tested it on a machine with Debian 11 and it works both over SSH and locally.

**Comment by tigerinus** [comment]
> I'm currently running official `v0.4.4`
> 
> ![image](https://user-images.githubusercontent.com/6792172/255337168-abadd161-cbb1-4ee3-a528-e9cee633d9e5.png)
> 
> and I'm getting Error 401 every time I access any of the commands: ![image](https://user-images.githubusercontent.com/6792172/255337206-78eee158-cf2b-4b4a-96b4-c8efbb0b9a9d.png)
> 
> Also already tried switching the port back to `80`, but still to no avail.

I will look into this issue.

Meanwhile, you should be able to manage appstores from UI now if it's official v0.4.4

**Comment by CorrectRoadH** [comment]
😗OK, I got it and fixed it in https://github.com/IceWhaleTech/CasaOS-Gateway/pull/37.

**Comment by rodrigofragadf** [comment]
> 😗OK, I got it and fixed it in [IceWhaleTech/CasaOS-Gateway#37](https://github.com/IceWhaleTech/CasaOS-Gateway/pull/37).

Updated and working fine! Thanks!
